### DISCUSSION TITLE
support naver login please

### DISCUSSION THREAD

**Discussion by smtSoon** [supabase/supabase#28435] [Feature Requests]
Our company's boss wants Naver login. If we can't create this service quickly, we'll be fired. Help, please!!!!!!!

**Comment by ekankr2** [comment]
I also wanted NAVER auth provider.

**Comment by gywlsdud05** [comment]
I would like to request the addition of Naver as an Auth Provider in Supabase. Naver is one of the most prominent IT companies in South Korea with a large user base. Adding Naver as an authentication provider would be beneficial for developers targeting the Korean market, allowing for easier integration and broader user accessibility.Thank you for considering this request.
